cuda_next/include/detail/dimensions.hpp

```diff
@@ -0,0 +1,109 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of CUDA Next in CUDA C++ Core Libraries,
+// under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES.
```

Comment:
```suggestion
// SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES.
```

I thoroughly hate dates in licenses